### PR TITLE
Bug fixes for language and access

### PR DESCRIPTION
**Pull request by obuisard**
Pull Request for Issue #47, #88, #89 .

### CODE REVIEW

File: administrator/components/com_guidedtours/forms/filter_tours.xml [hunk 1]
```diff
@@ -14,9 +14,26 @@
...
-			>
+		>
 			<option value="">JOPTION_SELECT_PUBLISHED</option>
 		</field>
+		<field
+			name="access"
+			type="accesslevel"
+			label="JGRID_HEADING_ACCESS"
+			onchange="this.form.submit();"
+		>
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/forms/filter_tours.xml]:
```suggestion
			>
```

File: administrator/components/com_guidedtours/forms/filter_tours.xml [hunk 2]
```diff
@@ -14,9 +14,26 @@
...
+			<option value="">JOPTION_SELECT_ACCESS</option>
+		</field>
+		<field
+			name="language"
+			type="contentlanguage"
+			label="JGRID_HEADING_LANGUAGE"
+			onchange="this.form.submit();"
+		>
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/forms/filter_tours.xml]:
```suggestion
			>
```

File: administrator/components/com_guidedtours/forms/filter_tours.xml [hunk 3]
```diff
@@ -27,20 +44,19 @@
 			onchange="this.form.submit();"
 			default="a.ordering ASC"
 			validate="options"
-			>
+		>
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/forms/filter_tours.xml]:
```suggestion
			>
```

File: administrator/components/com_guidedtours/src/Model/ToursModel.php [hunk 4]
```diff
@@ -57,13 +57,13 @@ public function __construct($config = [])
...
-    * @return DatabaseQuery
-    *
-    * @since 4.0.0
-    */
+    /**
+     * Provide a query to be used to evaluate if this is an Empty State, can be overridden in the model to provide granular control.
+     *
+     * @return DatabaseQuery
+     *
+     * @since 4.0.0
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/src/Model/ToursModel.php]:
```suggestion
     * @since __DEPLOY_VERSION__
```

File: administrator/components/com_guidedtours/src/Helper/StepHelper.php [hunk 5]
```diff
@@ -39,7 +39,7 @@ public static function getTourLanguage(int $id): string
             return "*";
         }
 
-        $db    = Factory::getDbo();
+        $db = Factory::getDbo();
```
Comment (brianteeman) [inline administrator/components/com_guidedtours/src/Helper/StepHelper.php]: please revert the change on this line. the codestyle was correct

**Comment by obuisard** [comment]
Thank you Aaron!
